cloud-sql/mysql/activerecord/app/assets/javascripts/cable.js

```diff
@@ -3,7 +3,6 @@
 //
 //= require action_cable
 //= require_self
-//= require_tree ./channels
```

Comment:
what is the reason for deleting this?

Reply:
This was also causing a error on deployment to App Engine.
Updating cable.js to match the Postgres sample resolved this error.